components/images.css

```diff
@@ -0,0 +1,20 @@
+
+
+.img-responsive{
+    object-fit: contain;
+    width: 80%;
```

Comment:
For a responsive image, you should give `width:100%`. Use this image inside a div and set the width and height property and the image will adapt.

Reply:
Required changes done